### ISSUE TITLE
Accessors of type Duration fields being flagged as EI

### ISSUE THREAD

**Issue by DominicAnelloSonos** [spotbugs/spotbugs#1634]
Spotbugs `4.3.0` is flagging getters of type **`java.util.Duration`** fields with `EI_EXPOSE_REP` and setters with `EI_EXPOSE_REP2` indicating that these methods expose mutable application state.  However **`Duration`** is immutable (all of its fields are final), so I believe this finding is in error.

Here's a minimal example of the issue.
```java
import java.time.Duration;

public class DurationBugTest {
    public static final Duration DEFAULT_D = Duration.ofSeconds(1L);
    private Duration d = DEFAULT_D;

    public Duration getD() {
        // flagged EI_EXPOSE_REP
        return this.d;
    }

    public void setD(Duration value) {
        // flagged EI_EXPOSE_REP2
        this.d = value;
    }
}
```

**Comment by gtoison** [comment]
This issue was fixed by #1677 (classes from the `java.time` package are considered immutable)
